ward/run.py

```diff
@@ -17,14 +18,54 @@
 from ward.config import set_defaults_from_config
 from ward.rewrite import rewrite_assertions_in_tests
 from ward.suite import Suite
-from ward.terminal import SimpleTestResultWrite, get_exit_code
+from ward.terminal import SimpleTestResultWrite, output_fixtures, get_exit_code
 
 init()
 
 sys.path.append(".")
 
 
-@click.command(context_settings={"max_content_width": 100})
+# TODO: simplify to use invoke_without_command and ctx.forward once https://github.com/pallets/click/issues/430 is resolved
+@click.group(
+    context_settings={"max_content_width": 100},
+    cls=DefaultGroup,
+    default="test",
+    default_if_no_args=True,
+)
+@click.pass_context
+def run(ctx: click.Context):
+    pass
+
+
+config = click.option(
+    "--config",
+    type=click.Path(
+        exists=False, file_okay=True, dir_okay=False, readable=True, allow_dash=False
+    ),
+    callback=set_defaults_from_config,
+    help="Read configuration from PATH.",
+    is_eager=True,
+)
+path = click.option(
+    "-p",
+    "--path",
+    type=click.Path(exists=True),
+    multiple=True,
+    is_eager=True,
+    help="Look for tests in PATH.",
+)
+exclude = click.option(
+    "--exclude",
+    type=click.STRING,
+    multiple=True,
+    help="Paths to ignore while searching for tests. Accepts glob patterns.",
```

Comment:
I'm suggesting we change the wording here because they're shared between the `test`/`fixture` commands. Otherwise the output of `ward fixtures --help` doesn't make sense.

```suggestion
    help="Paths to ignore while searching for items. Accepts glob patterns.",
```